packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -5687,7 +5687,7 @@ export interface IModalStyles {
 // @public (undocumented)
 export interface INav {
     focus(forceIntoFirstElement?: boolean): boolean;
-    selectedKey: string | undefined;
+    selectedKey: string | null | undefined;

```

Comment:
Can we avoid `null` as something we need to consider? I still recommend testing for `'selectedKey' in this.props` rather than allowing a specific `null` assignment.

Reply:
I previously requested that she *not* explicitly test for `'selectedKey' in this.props` because it's brittle and runs counter to the general pattern of handling explicit and implicit undefined the same way.

Can you explain more about why you're opposed to explicit null?

Reply:
This is almost a case where I would recommend using the empty string `''` instead of `null`. The issue is that `null` tends to cascade and break stuff everywhere because code now has to differentiate it from `undefined`.